tests/test_check_input.py

```diff
@@ -20,9 +20,9 @@ class TestClass():
     t_dirbamA=testdir+'testBamA'
     t_dirbamB=testdir+'testBamB'
     cwdpath=os.getcwd()
-    configdir = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'config/')
-    t_json=configdir+'fileTypes.json'
-   
+    configdir = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../archCompare/config/')
+    t_json=configdir+'compareMethods.json'
```

Comment:
Don't use the config dir stuff in a test.  A `json` files should exist in the `tests/data` tree for each specific thing they are testing (named appropriately).